index.js

```diff
@@ -23,6 +22,7 @@ module.exports = function (opt) {
 
     var options = merge({
       bare: false,
+      coffee: require('coffee-script'),
```

Comment:
We should if (1) the cost of parsing+executing coffee-script is high, or (2) requiring coffee-script has side effects. I don't think either of these are the case.

What do you think?